modules/core/core-api/src/main/java/com/enonic/xp/project/CreateProjectParams.java

```diff
@@ -75,6 +80,11 @@ public boolean isForceInitialization()
         return forceInitialization;
     }
 
+    public String getTimezone()
```

Comment:
```suggestion
    public TimeZone getTimeZone()
```

Reply:
After discuss we decided to use `ZoneId`